movie_theater/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for movie_theater project.
+
+Generated by 'django-admin startproject' using Django 5.0.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-xvh7oxs0t8$-new^b=7d$jg(x-pw-f1zxe)49zp@i*=+e&y^&e"
```

Comment:
The `SECRET_KEY` is hardcoded and exposed in the settings file. This is a security risk, especially in production environments. Consider using environment variables to securely manage the secret key.